doc/latest_news.html

```diff
@@ -4,6 +4,7 @@
       December 22nd 2015: HoloViews 1.4.1 released and now available on 
       <a href="https://pypi.python.org/pypi/holoviews/">PyPI</a> and
       <a href="https://anaconda.org/anaconda/holoviews">Anaconda</a>.
+      Now includes full support for the <a href="http://bokeh.pydata.org">Bokeh</a> plotting library.
```

Comment:
Well, what defines the list of "all elements"?  Can't the list of Bokeh elements be treated as the canonical ones, with the extra ones provided by Matplotlib as bonuses?  Seems like it's full support if we say it is. :-)  I'm reluctant to say "official", because that sounds like we have a team of paid support professionals at our disposal!  But whatever way you and Philipp want to say it is fine.  I do want to indicate that the Bokeh support is very different from the Plotly support -- the Plotly backend is *not* full support, right now at least.